test/rest_handler_SUITE.erl

```diff
@@ -474,6 +475,39 @@ delete_resource_missing(Config) ->
 	{response, _, 500, _} = gun:await(ConnPid, Ref),
 	ok.
 
+create_resource_created(Config) ->
+	doc("POST to an existing resource to create a new resource. "
+		"When the accept callback returns {created, NewURI}, "
+		"the expected reply is 201 Created."),
+	ConnPid = gun_open(Config),
+	Ref = gun:post(ConnPid, "/create_resource?created", [
+		{<<"content-type">>, <<"application/text">>}
+	], <<"hello">>, #{}),
+	{response, _, 201, _} = gun:await(ConnPid, Ref),
+	ok.
+
+create_resource_see_other(Config) ->
+	doc("POST to an existing resource to create a new resource. "
+		"When the accept callback returns {see_other, NewURI}, "
+		"the expected reply is 303 See Other."),
+	ConnPid = gun_open(Config),
+	Ref = gun:post(ConnPid, "/create_resource?see_other", [
+		{<<"content-type">>, <<"application/text">>}
+	], <<"hello">>, #{}),
+	{response, _, 303, _} = gun:await(ConnPid, Ref),
+	ok.
+
+create_resource_legacy(Config) ->
+	doc("POST to an existing resource to create a new resource. "
+		"When the accept callback returns {true, NewURI}, the legacy "
+		"behavior is a 302 See Other reply."),
+	ConnPid = gun_open(Config),
+	Ref = gun:post(ConnPid, "/create_resource", [
+		{<<"content-type">>, <<"application/text">>}
+	], <<"hello">>, #{}),
+	{response, _, 303, _} = gun:await(ConnPid, Ref),
+	ok.
```

Comment:
Do not test the old behavior. On the other hand it would be worth testing `true` and `false` return values if they aren't already.

Reply:
Ok.  I am used to document deprecated "stuff" and keep the tests, and then after N releases the deprecated "stuff" is removed, including the documentation and tests.  I think it is clear, but I will of course adapt to your policy.

`true` and `false` are already tested by `accept_callback_h.erl`

Reply:
Keep the tests yes, not add new ones. :-)

I only document what should be used, on the other hand I keep the documentation from previous releases with a version select so people can follow whichever they're currently using.